index.html

```diff
@@ -835,12 +835,14 @@ <h3>INR 2500</h3>
                             </table>
                           </div>
                       </div>
+                      <br>
+                      <p>Want to come for the conference, but not sure how to pitch the same to your boss. Leave it to us, check out <a href ="convince-my-boss.html"><strong><u>Convince My Boss.</u></strong></a></p>
```

Comment:
Add `text-center` and `pycon-volunteer` classes. Use italics instead of underline and remove the period.

Reply:
I will implement the same, I am out for a while. Till then then we can get other reviews.